apps/src/code-studio/projectRedux.js

```diff
@@ -87,6 +99,18 @@ export const setProjectUpdatedError = () => ({
   status: projectUpdatedStatuses.error
 });
 
+export const displayWorkspaceAlertOn = errorMsg => ({
```

Comment:
naming nit: I think it would be more Redux-y to give these more action oriented names, i.e. showWorkspaceAlert / hideWorkspaceAlert (or displayWorkspaceAlert, closeWorkspaceAlert, etc). And the state value "displayWorkspaceAlert" could be called something like "workspaceAlertVisible" or "displayingWorkspaceAlert", to more accurately capture what view state it's driving.

Reply:
Sounds good - will do!

Reply:
I like displayWorkspaceAlert and closeWorkspaceAlert with the state value displayingWorkspaceAlert.

Reply:
I'll change to showWorkspaceAlert and hideWorkspaceAlert with state value of showWorkspaceAlert to match the other actions and state values in the file.